Cython/Includes/cpython/list.pxd

```diff
@@ -29,12 +59,17 @@ cdef extern from "Python.h":
     Py_ssize_t PyList_GET_SIZE(object list)
     # Macro form of PyList_Size() without error checking.
 
+    PyObject* PyList_GetItemRef "__Pyx_CAPI_PyList_GetItemRef" (object list, Py_ssize_t index) except NULL
```

Comment:
This would probably be better as


```suggestion
    object PyList_GetItemRef "__Pyx_CAPI_PyList_GetItemRef" (object list, Py_ssize_t index)
```

It's an owned reference which is what Cython expects from a function returning `object`. It'll also do the `NULL` check automatically.

`PyList_GetItem` is declared with an `PyObject*` return type just because Cython doesn't know what to do with a borrowed reference,